### ISSUE TITLE
Build of Levure v0.9.3 network drive project fails with moving file error

### ISSUE THREAD

**Issue by jim1001** [trevordevore/levure#86]
My set up:
LC Community 8.1.6 on Windows 7
Levure v0.9.3
Levure sample project

The problem:
Build fails when project on network drive. 
build.log file says:
Done packaging application with error: moving folder [C:/Users/Jim/AppData/Local/Temp/revstandalone1502798507323/LevureStandalone] to [Z:/Temp/LevureSample/builds/development/0.0.1-1] (1)

The project builds successfully on the local C: drive.

build.log attached (with added .txt suffix to allow upload)
[build.log.txt](https://github.com/trevordevore/levure/files/1224960/build.log.txt)

**Comment by macMikey** [comment]
I think this is related to 84.  Trevor said this is an 8.1.6 problem slated to be fixed in 8.1.7.  Trevor, do you have the bug number so we can cc on that list?  The only one I have found that might be related is 17631.

**Comment by trevordevore** [comment]
@jim1001 Try replacing the packager/packager.livecodescript file in your levure folder with the attached version. revMoveFolder isn't providing a helpful error message. I've replaced it with my own version which should give us a more meaningful error message.

[packager.livecodescript.zip](https://github.com/trevordevore/levure/files/1225843/packager.livecodescript.zip)

**Comment by trevordevore** [comment]
@jim1001 I just tried a test similar to yours and I see the same error. I tested on Windows 10 running under VMWare Fusion. After replacing the packager.livecodescript with the version I posted above building a standalone worked. It looks like revMoveFolder has a bug in it. I'll update Levure to use my handler that moves folders.

**Comment by trevordevore** [comment]
I've posted the update to the `develop` branch. You can use the update I posted above until `develop` is merged into master and a new release is made. I'm almost done reworking the packager in order to support some features needed for a second project I am moving over to Levure. Once that is done I'll put out a new release.

**Comment by jim1001** [comment]
Trevor,

Thanks for the quick action. I tried the new packager with the sample project  on a network drive and the build worked as you said.
